mix.exs

```diff
@@ -33,6 +33,7 @@ defmodule Cloak.Mixfile do
     [
       {:ecto, ">= 1.0.0"},
       {:flow, "~> 0.13.0"},
+      {:pbkdf2, "~> 2.0", optional: true},
```

Comment:
Out of ease and "let's use something that has been tested before", I went with adding this Erlang library as an optional dependency. Are you okay with that?

Reply:
I try to be careful about adding core dependencies to the library, but this would be fine as an optional dependency, yes.